server/game/core/Game.js

```diff
@@ -83,17 +83,17 @@ class Game extends EventEmitter {
         Contract.assertArraySize(details.players, 2, 'Game must have exactly 2 players');
 
         details.players.forEach((player) => {
-            this.playersAndSpectators[player.user.username] = new Player(
-                player.id,
+            this.playersAndSpectators[player.user.id] = new Player(
+                player.user.id,
```

Comment:
In this particular context player is a combination of the user an their deck data. The user is going to be more information about their identity on the front end so we can easily match between a lobby user and their player assigned to them in game. I think right now since it's mostly just id and username that we're tracking it seems a bit gratuitous, but I think eventually this will be a good place for things like settings and social features.